nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/conf/logback.xml

```diff
@@ -89,7 +89,8 @@
     <logger name="org.apache.nifi.processors" level="WARN"/>
     <logger name="org.apache.nifi.processors.standard.LogAttribute" level="INFO"/>
     <logger name="org.apache.nifi.controller.repository.StandardProcessSession" level="WARN" />
-    
+
+    <logger name="org.apache.ignite" level="WARN"/>
```

Comment:
Instead of adding this to the default logback, which would get used for every NiFi instance, why not add documentation to the description or Additional Details section stating that periodic stats are logged and this statement, added to the logback.xml, will suppress them. 

This will get around adding a logger definition here that's specific to the processor but also give the user the ability to change it if needed. Besides, some users may like having the stats logged by default.